app/controllers/merchants/orders_controller.rb

```diff
@@ -1,4 +1,12 @@
 class Merchants::OrdersController < Merchants::BaseController
   def index
+    merchant = User.find(current_user.id)
+    @orders = Order.find_by_merchant(merchant)
+  end
+
+  def show
+    merchant = User.find(current_user.id)
+    @order = Order.find(params[:id])
+    @items = Item.find_by_order(@order, merchant)
```

Comment:
Maybe `find_by_order` could be an `Order` instance method that takes the merchant as an argument?

```
@items = @order.merchants_items(merchant)
```

Where:
```
class Order < Application Record
  #Code
  def merchants_items(merchant)
    items.where(user:merchant)
  end
end
```

My thinking is that we have already "pulled" out the order information, so we might as well use that in finding the items